pages/common/vue-serve.md

```diff
@@ -0,0 +1,8 @@
+# vue serve
+
+> A subcommand provided by `@vue/cli` and `@vue/cli-service-global` that allows you to do painless prototyping.
```

Comment:
This description is exactly the same as the one for `vue build`. What's the difference between the two subcommands?

Reply:
Build will create a production release in the `dist` directory, and serve sets up a hot-reloading development server.

Reply:
Can we tweak the descriptions to make this distinction explicit, then?

Reply:
Changes applied. Thanks for your suggestions!